playground/cloudflare-env/__tests__/custom-env/cloudflare-env.spec.ts

```diff
@@ -0,0 +1,6 @@
+import { expect, test } from 'vitest';
+import { getTextResponse } from '../../../__test-utils__';
+
+test('returns the correct custom-env var when CLOUDFLARE_ENV=custom-env', async () => {
+	expect(await getTextResponse()).toEqual('Custom env var');
+});
```

Comment:
I tried to use `vi.stubEnv` (https://vitest.dev/api/vi.html#vi-stubenv) here instead of changing `process.env` in the Vite config. It didn't work though (I think because the Vite config has already run by the time we run the test).